src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/ConsoleParameters.cs

```diff
@@ -48,7 +48,8 @@ public string LogFilePath
             set
             {
                 ValidateArg.NotNullOrEmpty(value, "LogFilePath");
-                if (!fileHelper.DirectoryExists(Path.GetDirectoryName(value)))
+                var directoryPath = Path.GetDirectoryName(value);
+                if ((directoryPath != string.Empty) && !fileHelper.DirectoryExists(Path.GetDirectoryName(value)))
```

Comment:
Yeah, definitely. I didn't realize Path.GetDirectoryName() could return null. God help anyone running the test platform on a root.